engine/src/main/java/org/terasology/rendering/opengl/LwjglFrameBufferObject.java

```diff
@@ -63,13 +63,6 @@ public LwjglFrameBufferObject(ResourceUrn urn, BaseVector2i size) {
             throw new IllegalStateException("Something went wrong with framebuffer! " + result);
         }
 
-        // clear and fill with full alpha
-        FloatBuffer buffer = BufferUtils.createFloatBuffer(16);
-        GL11.glGetFloat(GL11.GL_COLOR_CLEAR_VALUE, buffer);
-        GL11.glClearColor(0f, 0f, 0f, 1f);
-        GL11.glClear(GL11.GL_COLOR_BUFFER_BIT);
-        GL11.glClearColor(buffer.get(), buffer.get(), buffer.get(), buffer.get());  // reset
-
         GL30.glBindFramebuffer(GL30.GL_FRAMEBUFFER, 0);
```

Comment:
P.S. I'm just noticing there are other lines using OpenGL 3.0. I guess to convert everything to 2.1-compatible code is beyond the scope of this PR.